src/components/shared/select/Select.tsx

```diff
@@ -0,0 +1,24 @@
+import "./Select.scss";
+
+export interface SelectProps {
+  selectName: string;
+  data: string[];
+}
+
+const Select = ({ selectName, data }: SelectProps) => {
```

Comment:
Why didn't you use select from material ui?

Reply:
will fix it in next PR.